functions/_fzf_search_shell_variables.fish

```diff
@@ -1,20 +1,15 @@
-function __fzf_search_shell_variables --description "Search and inspect shell variables using fzf. Insert the selected variable into the commandline at the cursor."
+function _fzf_search_shell_variables -d "Search and inspect shell variables using fzf. Insert the selected variable into the commandline at the cursor."
```

Comment:
Why switch to the short option? My rule of thumb is to always use the long form of options when scripting and use short hand when using the shell interactively.

Reply:
You're not wrong. I'd agree that long options _are_ better for scripting in general. But I personally make an exception for the the most insanely common options like `set -l` or `functions -d`. You may feel differently and choose to be 100% consistent, but I think it's better to just save the line length since everyone contributing _should_ know these ultra basic options. To me it's like `rm -r`.